AlphaWallet/WalletConnect/WalletConnectProvider.swift

```diff
@@ -258,6 +264,14 @@ extension WalletConnectProvider: WalletConnectServerDelegate {
             .eraseToAnyPublisher()
     }
 
+    private func verifyIfEip712v3And4(_ typedData: EIP712TypedData, walletSession: WalletSession) throws -> EIP712TypedData {
```

Comment:
And like the other `verifyIfEip712v3And4` (for dapp browser), better to return `Bool` + functional?